### PR TITLE
Fix cross version tests

### PR DESCRIPTION
**Pull request by blindpirate**
### Context

This PR is to fix several cross version tests which would fail Master Accept.

### CODE REVIEW

File: subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r43/ToolingApiVersionSpecification.groovy [hunk 1]
```diff
@@ -44,10 +44,6 @@ abstract class ToolingApiVersionSpecification extends ToolingApiSpecification {
         return "Support for builds using Gradle older than 2.6 was deprecated and will be removed in 5.0. You are currently using Gradle version ${version}. You should upgrade your Gradle build to use Gradle 2.6 or later."
     }
 
-    String getOutput() {
```
Comment (oehme) [inline subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r43/ToolingApiVersionSpecification.groovy]: Why was this removed?

Comment (blindpirate) [reply subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r43/ToolingApiVersionSpecification.groovy]: It will fail the Master Accept on CI in this line: https://github.com/gradle/gradle/blob/1af8d212e94f1c098abb26190d4e0f0d2823211b/subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r43/ToolingApiVersionSpecification.groovy#L51

Last night I noticed this failure and I decided to make a quick fix in case the failure blocks someone else's build, so I simply removed it.

It has been added back here: 
https://github.com/gradle/gradle/pull/2952/files#diff-a5fe5712c68e986ad50d9fa18a0ad8f8R35
